YouTube Livestream Botter.py

```diff
@@ -36,75 +42,78 @@ def __init__(self):
         self.botted = 0
         self.combolen = self.combolist.qsize()
 
-    def printservice(self): #print screen
+    def printservice(self):  # print screen
         while True:
             if True:
                 os.system(clear)
                 print(Fore.LIGHTCYAN_EX + intro + Fore.LIGHTMAGENTA_EX)
-                print(
-                    Fore.LIGHTCYAN_EX + f"Botted:{self.botted}\n")
+                print(Fore.LIGHTCYAN_EX + f"Botted: {self.botted}")
                 for i in range(len(self.printing) - 10, len(self.printing)):
                     try:
                         print(self.printing[i])
                     except (ValueError, Exception):
                         pass
                 time.sleep(0.5)
+
+
 a = main()
+
+
 class proxy():
 
     def update(self):
         while True:
-            url = "https://api.proxyscrape.com/?request=getproxies&proxytype=http&timeout=10000&ssl=yes"
+            url = "https://api.proxyscrape.com/?request=getproxies&proxytype=http&timeout=1000&ssl=yes"
```

Comment:
what's the point of setting the proxy response time to 1000 milliseconds?

Reply:
If you set it too low, there is a chance of not getting any proxies at all. Because they are free proxies, they often don't have the best Quality, so you just take whatever proxies you can get your hands on.